dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -5076,12 +5076,12 @@ html, body {
 }
 
 .dijitMenu .dijitMenuItemHover {
-  background-color: #eeeeee;
```

Comment:
#eee does not exist on https://github.com/dotCMS/dotcms-dojo-theme/blob/da7e41078cf79838a14e526e9b8390b89ab2452f/scss/dijit/_variables.scss#L1